elide-core/src/main/java/com/yahoo/elide/parsers/state/RecordTerminalState.java

```diff
@@ -49,6 +55,14 @@ public RecordTerminalState(PersistentResource record) {
     }
 
     @Override
+    public Supplier<Pair<Integer, JsonNode>> handlePost(StateContext state) {
+        if (collectionTerminalState.isPresent()) {
+            return collectionTerminalState.get().handlePost(state);
+        }
+        throw new IllegalStateException("Cannot POST to a record.");
```

Comment:
Should we make a new Illegal Request exception to return error code instead of 500?